### DISCUSSION TITLE
Entity Linker

### DISCUSSION THREAD

**Discussion by shrinidhin** [explosion/spaCy#10778] [Help: Coding & Implementations] [answered]
Hello!So I have been using an Entity Linker to link entities and resolve ambiguities in news articles about stocks. I have a few questions which I hope can be addressed:
1. While I understanding adding aliases is important at the beginning, Shouldn't this be in some way trainable?As in given the kb_ids in the data along with the references, The model learns all possible aliases?Maybe the model computes or identifies the most probable entity using similarity score or something?Because if every possible alias has to be listed for each entity, Then how is it better than a non-ML model? 
For e.g. In my use-case, For the Entity `Dr. Lal Pathlabs`, I have added the alias `Dr. Lal`, But if in a piece of text I have `Dr Lal`, then the kb_id for that is still null cos of the missing `.`
2. Also, I noticed that the aliases are case sensitive. For eg: For an Entity like `Nifty 50`, `Nifty` and `nifty` are to be given as two different aliases. Can this be modified in any way?Because it seems very tedious if it is case sensitive.
Thank you!

**Comment by svlandeg** [comment, marked as answer]
Duplicate of https://github.com/explosion/spaCy/discussions/10713. Please follow up on the other thread to keep the discussion in one place.
